GraphicEditor/MapObject.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+namespace GraphicEditor
+{
+    class Rectt
+    {
+        public String Id { get; set; }
+        public String RoomType { get; set; }
+        public int Width { get; set; }
+        public int Height { get; set; }
+        public int Left { get; set; }
+        public int Top { get; set; }
+        public SolidColorBrush Color { get; set; }
+    }
+    class MapObject
+    {
+
+        public MapObject()
+        {
+
+        }
+
+        public SolidColorBrush Border { get; private set; }
+
+        public List<Rectt> LoadFromFile(string textFile)
+        {
+            List<Rectt> rectangles = new List<Rectt>();
+
+            if (File.Exists(textFile))
+            {
+                // Read a text file line by line.  
+                string[] lines = File.ReadAllLines(textFile);
+                foreach (string line in lines)
+                {
+                    string[] parts = line.Split(',');
+                  //  byte b1 = Byte.Parse(parts[6]);
+                   // byte b2 = Byte.Parse(parts[7]);
+                    //byte b3 = Byte.Parse(parts[8]);
+
+                    //Color col = Color.FromRgb(b1, b2, b3);
+
+                    rectangles.Add(new Rectt()
+                    {
+                        Id = parts[0],
+                        RoomType = parts[1],
+                        Width = Int32.Parse(parts[2]),
+                        Height = Int32.Parse(parts[3]),
+                        Left = Int32.Parse(parts[4]),
+                        Top = Int32.Parse(parts[5])
```

Comment:
I suggest you to add one more constructor with fields for Rectt class,so you can assign all these values in one line,so it will be much more clean.Also suggest you to change name to Rectangle,it will be more readable

Reply:
agree